### PR TITLE
feat: add dice icons

### PR DESCRIPTION
**Pull request by mittalyashu**
| Preview |
| --- |
| CleanShot 2022-03-10 at 23 18 45@2x |

Reference: https://github.com/lucide-icons/lucide/discussions/463

### CODE REVIEW

**Comment by mittalyashu** [comment]
My first thoughts are the dice (1-6) after quite thin. What do you guys think?

**Comment by ericfennis** [comment]
@mittalyashu Nice icons, yeah the dots lookes a bit thin.

Did you saw @johnletey s icon from feather [#918](https://github.com/feathericons/feather/pull/918/files?short_path=3795b10#diff-3795b1030c559e2db335c718b2c7ad01ab25caf531433fdda69dae4212e13d66)?

**Comment by mittalyashu** [comment]
@ericfennis I don't see much difference except the width/height is 18px. 🤔

**Comment by ericfennis** [comment]
Width of the box needs to be 18 to match with the rectangle icon.

Maybe use the dots from `divide`?

**Comment by mittalyashu** [comment]
Using the dot from `divide` will not work either 😂 

CleanShot 2022-02-04 at 21 02 03@2x

**Comment by ericfennis** [comment]
@mittalyashu Ok never mind, the other ones looks better 😆 .
But keep the 18 px rectangle, I think that will look fine.

File: icons/dice-1.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 4.8A1.8 1.8 0 0019.2 3H4.8A1.8 1.8 0 003 4.8v14.4A1.8 1.8 0 004.8 21h14.4a1.8 1.8 0 001.8-1.8V4.8z" />
```
Comment (ericfennis) [inline icons/dice-1.svg:12]:
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

File: icons/dice-2.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 4.8A1.8 1.8 0 0019.2 3H4.8A1.8 1.8 0 003 4.8v14.4A1.8 1.8 0 004.8 21h14.4a1.8 1.8 0 001.8-1.8V4.8z" />
```
Comment (ericfennis) [inline icons/dice-2.svg:12]:
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

File: icons/dice-3.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 4.8A1.8 1.8 0 0019.2 3H4.8A1.8 1.8 0 003 4.8v14.4A1.8 1.8 0 004.8 21h14.4a1.8 1.8 0 001.8-1.8V4.8z" />
```
Comment (ericfennis) [inline icons/dice-3.svg:12]:
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

File: icons/dice-4.svg [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 4.8A1.8 1.8 0 0019.2 3H4.8A1.8 1.8 0 003 4.8v14.4A1.8 1.8 0 004.8 21h14.4a1.8 1.8 0 001.8-1.8V4.8z" />
```
Comment (ericfennis) [inline icons/dice-4.svg:12]:
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

File: icons/dice-5.svg [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 4.8A1.8 1.8 0 0019.2 3H4.8A1.8 1.8 0 003 4.8v14.4A1.8 1.8 0 004.8 21h14.4a1.8 1.8 0 001.8-1.8V4.8z" />
```
Comment (ericfennis) [inline icons/dice-5.svg:12]:
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

File: icons/dice-6.svg [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21 4.8A1.8 1.8 0 0019.2 3H4.8A1.8 1.8 0 003 4.8v14.4A1.8 1.8 0 004.8 21h14.4a1.8 1.8 0 001.8-1.8V4.8z" />
```
Comment (ericfennis) [inline icons/dice-6.svg:12]:
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]

**Comment by ericfennis** [comment]
@mittalyashu Thanks for changing, can you add some tags as well?

**Comment by Bowero** [comment]
Screenshot 2022-03-19 at 15 09 43

I made a version with thicker dots. Personally I think that fits the style better.

Also, a proposal for the tags:

```json
"dice-1": [
  "dice",
  "random",
  "tabletop"
],
"dice-2": [
  "dice",
  "random",
  "tabletop"
],
"dice-3": [
  "dice",
  "random",
  "tabletop"
],
"dice-4": [
  "dice",
  "random",
  "tabletop"
],
"dice-5": [
  "dice",
  "random",
  "tabletop"
],
"dice-6": [
  "dice",
  "random",
  "tabletop"
],
```

**Comment by mittalyashu** [comment]
@Bowero We've to keep the width and height of the square at `18px`.

**Review by ericfennis — APPROVED** [review]

**Comment by Spleefies** [comment]
I think that the version with bigger dots looks better.

**Pull request merged**